src/language/__tests__/schema-kitchen-sink.graphql

```diff
@@ -17,6 +17,7 @@ type Foo implements Bar {
   four(argument: String = "string"): String
   five(argument: [String] = ["string", "string"]): String
   six(argument: InputType = {key: "value"}): Type
+  seven(argument: Int = null): Type
```

Comment:
This might have been discussed elsewhere, but I'm confused why we want to support `null` as a **default** value.

How is this different from:

```graphql
seven(argument: Int): Type
```

Except that one will have `args["argument"]` set to `null` and the other `args["argument"]` will be undefined.

cc @rmosolgo